extensions/roc-plugin-test-jest/src/config/roc.config.js

```diff
@@ -2,8 +2,10 @@ export default {
     settings: {
         test: {
             jest: {
-                reporter: '',
-                reportpath: 'junit.xml',
+                junit: {
+                    enabled: false,
+                    reportpath: 'junit.xml',
```

Comment:
Should we have a convention here on placing the report in some directory? Maybe a reports folder?